style.css

```diff
@@ -0,0 +1,102 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap");
+
+body {
+  display: block;
+  margin: 0;
+}
```

Comment:
Optional suggestions

- [ ] Kindly, add the `overflow-x:hidden` to remove the scroll left of the page.

Reply:
Thank you for your professional review.

Reply:
Nice Job BTW, all suggestions has been solved.